app/app.rb

```diff
@@ -0,0 +1,51 @@
+ENV['RACK_ENV'] ||= 'development'
+require 'sinatra/base'
+require 'sinatra/flash'
+require './app/models/data_mapper_setup'
+require './app/models/peep'
+require './app/models/user'
+
```

Comment:
Did you get onto the part of the challenge (during the week) where routes for different resources are split out into separate controllers?  This is called the modular pattern and, if you've not already read about it and tried it out, I recommend that you give it a shot.  We can talk about this if you like.

Also, have a quick read about what REST means and how to make your routes RESTful.  For example:

```ruby
# this
get 'posts/new' do
    <code>
end

# rather than this
get '/new' do
    <code>
end
```